src/components/header.tsx

```diff
@@ -83,25 +84,30 @@ export default function Header() {
   const button = getHeaderButton();
 
   return (
-    <Wrap isScrolled={isScrolled}>
-      {checkIsHome() ? (
-        <Hamburger onClick={() => setIsToggleHamburger(true)} />
-      ) : (
-        <BackButton
-          onClick={() => {
-            window.history.back();
-          }}
-        />
-      )}
-      {isToggleHamburger ? (
-        <Navigation isNaviOpened={isToggleHamburger} onRequestClose={() => setIsToggleHamburger(false)} />
-      ) : null}
-      {title && <Title>{title}</Title>}
-      {button && (
-        <OptionalButton to={button.link}>
-          <ButtonLabel>{button.label}</ButtonLabel>
-        </OptionalButton>
-      )}
-    </Wrap>
+    <>
+      <Wrap isScrolled={isScrolled}>
+        {checkIsHome() ? (
```

Comment:
```suggestion
        {isHome() ? (
```
`check`와 `is~`라는 의미가 중복인 것 같은데, `isHome`으로 충분하지 않을까유???

Reply:
5663e2b 커밋에 반영되었습니다-!